notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>In other rows you output the dataframe head but here you do that info? Any particular reason for that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>This was from some debugging I was doing, but I can display it as head if it looks cleaner.</p>